docs/reference/feedstock-settings.md

```diff
@@ -0,0 +1,33 @@
+# Feedstock settings
+
+When `conda-smithy` rerenders the contents of the feedstock, it takes into account different sources of configuration:
```

Comment:
> `conda-smithy`

Link to list of repos/tools

Reply:
link to its reference on [tooling-data.md](https://github.com/Quansight-Labs/cf-infra-docs/pull/2/files#diff-467cf5c6d44651073fb85ecdc393c674bdd2b2c80adebc71b7cda271bbe8ec23)